apps/acrs/behaviours/aggregator-save-update.js

```diff
@@ -1,3 +1,5 @@
+// let isUpdate;
+// let updatingIndex;
```

Comment:
Can remove these commented out vars now session is used

Reply:
i forgot that gift there